api/libproject/devstart.rb

```diff
@@ -2604,8 +2618,17 @@ def get_leo_api_url(project)
   return get_fc_config(project)["leoBaseUrl"]
 end
 
-def get_auth_domain(project)
-  return get_fc_config(project)["registeredDomainName"]
+def get_access_tier_config(project, tier)
```

Comment:
nit: may be clearer to rename `tier` -> `tier_name` or `tier_short_name`, then can use `tier` instead of `myTier` below